src/modules/profile/Profile.tsx

```diff
@@ -0,0 +1,210 @@
+import { useAppContext } from 'src/context/AppContext';
+import styled from 'styled-components';
+import PermIdentityIcon from '@mui/icons-material/PermIdentity';
+import { Typography } from '@mui/material';
+import EditOutlinedIcon from '@mui/icons-material/EditOutlined';
+import AccountBalanceWalletIcon from '@mui/icons-material/AccountBalanceWallet';
+import SettingsIcon from '@mui/icons-material/Settings';
+import IosShareIcon from '@mui/icons-material/IosShare';
+import LogoutIcon from '@mui/icons-material/Logout';
+import { colors } from 'src/styles/colors';
+
+const StyledDivProfile = styled('div')`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #f6f6f6;
+  height: 100vh;
+`;
+const StyledDivUserDetailsSection = styled('div')`
+  width: 100%;
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  margin: 30px;
+`;
+const StyledDivProfilePicture = styled('div')`
+  border: 2px solid #ad00ff;
+  border-radius: 50%;
+  padding: 3px;
+  width: 80px;
+  height: 80px;
+`;
+const StyledDivUserInfo = styled('div')``;
+const StyledDivEditProfileButton = styled('button')`
+  margin: 10px;
+  color: #fff;
+  background-color: transparent;
+  border: none;
+
+  &:hover {
+    cursor: pointer;
+    background-color: #f1f1fa;
+  }
+`;
+const StyledDivAccountButtonGroup = styled('div')`
+  width: 90%;
+  background-color: #fff;
+  border-radius: 20px;
+  margin: 20px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 10px;
+`;
+
+const StyledAccountButtons = styled('button')`
+  display: flex;
+  align-items: center;
+  width: 100%;
```

Comment:
When you remove `align-items: center;` from the `StyledDivAccountButtonGroup`, you can remove `width: 100%;` as it's not needed.